scripts/dump_meta_data.sql

```diff
@@ -0,0 +1,114 @@
+-- This script will dump relevant meta data from internal TimescaleDB tables
+-- that can help our engineers trouble shoot.
+--
+-- usage:
+-- psql [your connect flags] -d your_timescale_db < dump_meta_data.sql > dumpfile.txt
+
+\echo 'TimescaleDB meta data dump'
+\echo '<exclude_from_test> Date, git commit, and extension version can change without it being an error.'
```

Comment:
I think having this in output is awkward. Can we instead check for a pset variable(akin to DISABLE_OPTIMIZATION and not output this if it's set or something?

Reply:
I agree that it is a bit awkward but I think it is worth it. I did it like this as I would like it to run to at least catch if it throws an error. Basically I want to test that it produces some output but I want it excluded from the diff.

Reply:
can't we `\o /dev/null` then to hide output if flag present but run the command?